sos/report/plugins/boot.py

```diff
@@ -56,5 +56,7 @@ def setup(self):
                 self.add_cmd_output(f"lsinitrd {image}", priority=100)
                 self.add_cmd_output(f"lsinitramfs -l {image}", priority=100)
 
+        # Capture Open Virtual Machine Firmware information
+        self.add_copy_spec("/sys/firmware/efi/ovmf_debug_log")
```

Comment:
I can move this capture to lines 33/34 if preferred. I thought it may be better to have this separated for clarity.

Reply:
You can add the same comment if you would merge this a_c_s with the one on lines 33/34 to keep sufficient clarity. If this piece of data is logically a lot of different, having extra a_c_s is reasonable - either works fine for me.